app/Http/Controllers/Api/V1/CustodianHasProjectOrganisationController.php

```diff
@@ -97,6 +104,11 @@ public function index(Request $request, int $custodianId)
                         $q->where('id', $projectId);
                     });
                 })
+                ->join('project_has_organisations', 'custodian_has_project_has_organisation.project_has_organisation_id', '=', 'project_has_organisations.id')
+                ->join('projects', 'project_has_organisations.project_id', '=', 'projects.id')
+                ->join('organisations', 'project_has_organisations.organisation_id', '=', 'organisations.id')
+                ->applySorting()
+                ->select('custodian_has_project_has_organisation.*')
```

Comment:
This is where we start needing to think about rawSQL. Eloquent is fine for surface level stuff, but its dog slow when joining tables.